CONTRIBUTING.rst

```diff
@@ -67,6 +67,20 @@ Install Flocker's development dependencies in a ``virtualenv`` by running the fo
 .. _Docker: https://www.docker.com/
 
 
+Centos-7
+^^^^^^^^
```

Comment:
This title should be CentOS 7 (the current style is causing a spelling related build break)